apps/prairielearn/src/pages/enroll/enroll.ts

```diff
@@ -109,6 +108,7 @@ router.post('/', [
       // Abuse the middleware to authorize the user for the course instance.
       req.params.course_instance_id = course_instance.id;
       await authzCourseOrInstance(req, res);
+      // Now we have authzData
```

Comment:
Nit, should this be using `buildAuthzData` instead of using the middleware?

Reply:
Should I be changing this as part of this PR?